docs/docs/versions/v0_3/index.mdx

```diff
@@ -78,6 +78,7 @@ well as updating the `langchain_core.pydantic_v1` and `langchain.pydantic_v1` im
 | langchain-mistralai                    | 0.2.0   | >=0.2,&lt;0.3                 |
 | langchain-mongodb                      | 0.2.0   | >=0.2,&lt;0.3                 |
 | langchain-nomic                        | 0.1.3   | >=0.1.3,&lt;0.2               |
+| langchain-nvidia                       | 0.3.0   | >=0.3.0,&lt;0.2               |
```

Comment:
please double check the constraint

Reply:
@mattf can you recheck this.
I guess the intention is we support langchain v3 from latest langchain-nvidia version and will support till 0.4 
which we can update later as future releases happen